api/src/main/java/io/opentelemetry/trace/DefaultTracer.java

```diff
@@ -35,7 +35,7 @@ public static Tracer getInstance() {
 
   @Override
   public Span getCurrentSpan() {
-    return TracingContextUtils.getCurrentSpan();
+    return Span.current();
```

Comment:
Given that `TracingContextUtils` _still_ is the root, I'd be happier with using that one instead (users and instrumentation are free to use whatever they want of course ;) )

Reply:
this is gone from the Tracer now, so probably not relevant. :)

Reply:
Disagree - we want to use whatever the root is. Otherwise, let's simply move the logic to `Span` ;) (although the Spec may not allow that).

Reply:
I'm just saying that this method is no longer on the Tracer on the main branch, so this comment can't apply without adding it back to the Tracer.